src/ServiceClient/Implements/DeribitServiceClient.cs

```diff
@@ -63,7 +70,35 @@ public async Task<bool> IsDeribitAvailableAsync(CancellationToken cancellationTo
                 var result = await socketDataTransfer.ReceiveAsync(socket, cancellationToken);
                 return result != null;
             }
-            catch (Exception)
+            catch
+            {
+                return false;
+            }
+        }
+
+        public async Task<bool> AuthenticateAsync(CancellationToken cancellationToken)
+        {
+            try
+            {
+                var request = new Request
+                {
+                    Id = AUTHENTICATE,
```

Comment:
_Id_ should be an autoincrement value in every request. There must be a generator